xtra-answers.md

```diff
@@ -0,0 +1,15 @@
+# *Java: The Complete Answers, Twelfth Edition* <sup>(Prophet Newspaper/Wambeek Home Edition, 2015)</sup>
+
+## 10. Continued
+
+An interface with an @ symbol that isn't really an interface.
+
+## 11. What is a marker annotation?
+
+An annotation purely to mark a point in the code.
```

Comment:
Almost - it's an annotation that contains no fields, meant purely to indicate something (such as _this method `@Override`s something from a supertype_).